server/sdc_db/seed.js

```diff
@@ -0,0 +1,26 @@
+const faker = require('faker');
+const fakeData = require('./fakeData.js');
+const fs = require('fs');
+
+console.time('seed')
+var data = 'host,city,country,state,state_abbrev,longitude,latitude,description,guests,bedrooms,baths,beds,price,rating\n';
+
+const outputFile = fs.createWriteStream('output.csv');
+
+outputFile.write(data, (err) => {
+  if (err) {
+    console.log(err.message);
+  }
+});
+
+for (let i = 0; i < 100; i++) {
```

Comment:
100!!?!? What happened to 10 million!?

Reply:
Haha! For testing purposes I changed this back to 100 to save time. I plan to compare csv and json creation times!